src/Domain.LinnApps/Boms/BomReportsService.cs

```diff
@@ -666,5 +666,155 @@ public ResultsModel GetBomDifferencesReport(string bom1, string bom2)
             reportLayout.SetGridData(values);
             return reportLayout.GetResultsModel();
         }
+
+        public ResultsModel GetBoardComponentSummaryReport(string boardCode, string revisionCode)
```

Comment:
Didn't Petra want the ability to print out components for a specified layout? I guess not if she's happy with what you've shown her

Reply:
Wait, you specify the layout as part of the revision code right? just ignore this comment